.editorconfig

```diff
@@ -99,17 +102,38 @@ dotnet_style_collection_initializer = true:suggestion
 dotnet_style_coalesce_expression = true:suggestion
 dotnet_style_null_propagation = true:suggestion
 dotnet_style_explicit_tuple_names = true:suggestion
+dotnet_style_readonly_field = true:suggestion
+dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggestion
+dotnet_style_prefer_inferred_tuple_names = true:suggestion
+dotnet_style_prefer_inferred_anonymous_type_member_names = true:suggestion
+dotnet_style_prefer_conditional_expression_over_return = true:refactoring
+dotnet_style_prefer_conditional_expression_over_assignment = true:refactoring
+dotnet_style_prefer_auto_properties = true:suggestion
+csharp_prefer_simple_default_expression = true:suggestion
+
+dotnet_code_quality_unused_parameters = non_public:suggestion
+dotnet_code_quality.ca1802.api_surface = private, internal
+dotnet_code_quality.CA2208.api_surface = public
 
 # CSharp code style settings:
 [*.cs]
 
 csharp_preserve_single_line_blocks = true
 csharp_preserve_single_line_statements = false
+csharp_prefer_braces = true:refactoring
```

Comment:
What is refactoring here?
Can you point a description?
I'd expect `true`.
https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-language-conventions?view=vs-2019#code-block-preferences

Reply:
It seems `refactoring` is not valid in this context. In other contexts it is a symonym for `silent`
https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-naming-conventions?view=vs-2019#severity

Reply:
issue opened upstream: https://github.com/dotnet/runtime/pull/37367